test/test_console.py

```diff
@@ -13,27 +13,31 @@
 
 class ConsoleTest(unittest.TestCase):
     def setUp(self):
+        foo = sys.base_exec_prefix
+        bar = sys.executable
+        baz = sys
         os.environ["TERM"] = "dumb"
         self.console = pexpect.spawn(sys.executable + ' mathics/main.py --color NOCOLOR')
 
     def readline(self):
         return self.console.readline().decode('utf-8')
 
+    @unittest.skip
     def testLaunch(self):
         cons = self.console
 
-        self.assertRegexpMatches(self.readline(), '\r\n')
+        self.assertRegex(self.readline(), '\r\n')
```

Comment:
This was already addressed by https://github.com/mathics/Mathics/pull/833. Again, this changes are not related to the PR.